src/python/TaskWorker/Actions/DBSDataDiscovery.py

```diff
@@ -247,6 +272,8 @@ def execute(self, *args, **kwargs):
     config.TaskWorker.cmskey = os.environ["X509_USER_KEY"]
 
     config.TaskWorker.DDMServer = 'dynamo.mit.edu'
+    config.TaskWorker.resturl = 'cmsweb.cern.ch'
```

Comment:
I think the TW config defines if TW will talk to prod cmsweb or testbed one, or even private instance for testint no ? There should not be hardcoded URL's in the code.

Reply:
Yes but those variables are needed only for the standalone execution of the module, they do not overwrite the TW config AFAICT.

Reply:
ah.. right. in this "snippet view" I midded the context (those lines are part of main, i.e. test)